docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -700,6 +700,14 @@ In some scenarios is required to have different values. To allow this we provide
 - `nginx.ingress.kubernetes.io/proxy-next-upstream-tries`
 - `nginx.ingress.kubernetes.io/proxy-request-buffering`
 
+If you indicate [Backend Protocol](#backend-protocol) as `GRPC` or `GRPCS`, the following values will be applied to gRPC connection as well:
```

Comment:
can you define the defaults or point to the docs of them please :) users may not know what {{ proxy_connect_timeout }} means. Thanks!!!

Reply:
Or maybe just point to nginx docs to the defaults and say they can be configured somewhere else

Reply:
Yeah, please point to the docs.nginx. thanks

Reply:
sure, i added the default values (same as proxy_*_timout) and nginx docs